nv_exporter.py

```diff
@@ -414,13 +415,14 @@ def collect(self):
             ifile_name_list = []
 
             for c in json.loads(response.text)['incidents']:
+                iidlist.append(c['id'])
                 if 'workload_name' in c:
                     itimelist.append(c['reported_timestamp'])
                     inamelist.append(c['name'])
```

Comment:
Could we move the line into the if clause, so its index will still align with other slices when `workload_name` is empty? 
```suggestion
                if 'workload_name' in c:
                    iidlist.append(c['id'])
                    itimelist.append(c['reported_timestamp'])
                    inamelist.append(c['name'])
```

Reply:
Hi ! just to understand, in which case might workload_name be empty? Shouldn't we change this if to check for the id variable?, which, by its nature, will always be present for each generated event?